random-menu.rb

```diff
@@ -0,0 +1,65 @@
+# Prints a welcome message
+puts "Welcome to my random menu generator! "
+
+# Defines three arrays whose elements are foods or food descriptors
+adjs = [
+  "creamed",
+  "warm",
+  "spicy",
+  "steamed",
+  "seasoned",
+  "barbecued",
+  "chilled",
+  "peppered",
+  "baked",
+  "steamed"
+]
+
+key_ingredients = [
+  "panceta",
+  "lentil",
+  "strawberry",
+  "shiitake",
+  "jalapeno",
+  "corn",
+  "tofu",
+  "chicken",
+  "bacon",
+  "lavender"
+]
+
+dishes = [
+  "soup",
+  "curry",
+  "salad",
+  "cake",
+  "creme brulee",
+  "meatloaf",
+  "chili",
+  "pizza",
+  "sushi",
+  "pancakes"
+]
+
+# Randomly rearranges the order of the elements in each array
+adjs = adjs.shuffle
```

Comment:
Nice use of shuffle! This is a clever way to get a random order without destroying the array as you go along.